detailed_docs/iAP_over_BT/iAP_over_BT_re-connection_use_case.md

```diff
@@ -0,0 +1,88 @@
+## Use Case 1: iAP over BT re-connection
+
+**Main Flow:**  
+
+_Pre-conditions:_  
+a. iOS device is connected over Bluetooth  
+b. Application from iOS device is registered and running on SDL 
+
+_Steps:_    
+1. User connects the same iOS device over USB  
+
+_Expected:_  
+
+2. SDL receives signal with `<UUID>` about USB connection of the same device 
+3. SDL compares `<UUID>` of BT device connection with `<UUID>` of USB connection  
+4. `<UUID>` of BT device connection matches with `<UUID>` of USB connection
+5. SDL sends signal about transport switch to the System  
+6. SDL starts reconnection timer
+7. SDL receives `<RegisterAppInterface>` request **before** reconnection timer expires
+8. Application sends **valid** "hashID" with `RegisterAppInterface` request 
+9. SDL sends succesful response to the application 
+10. SDL updates data in PolicyTable 
+11. SDL notifies HMI via `UpdatadeviceList` about changed `transportType`
+12. SDL does not notify HMI about new registration
+
+
+_Exception 1:_  
+7.1. **After** timer expires application sends **valid** "hashID" with `<RegisterAppInterface>` request  
+7.1.a. SDL notifies HMI about app being unregistered   
+7.1.b. Application registers with **valid** "hashID"  
+7.1.c. SDL performs data resumption for this application  
+7.1.d. SDL **notifies** HMI about fresh application registration 
+
+_Exception 2:_  
```

Comment:
What is the difference in case between Exception 2 and Exception 3?

Reply:
_Exception 2_ describes the flow when invalid or omitted hashID is received **after timer expires**;  
_Exception 3_ describes the flow when invalid or omitted hashID is received **before timer expires**